docs/src/app/versions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Humm, that sounds far too complicated. What about killing this `versions.jsx`? Do we really need this `DropDownMenu` for older versions? I don't think so.